qa/suites/rados/singleton-nomsgr/all/pool-access.yaml

```diff
@@ -0,0 +1,14 @@
+roles:
+- [mon.a, osd.0, osd.1, client.0]
+tasks:
+- install:
+- ceph:
+    conf:
+      osd:
+# we may land on ext4
+        osd max object name len = 400
+        osd max object namespace len = 64
```

Comment:
you might want to backport 85e2f3f31d25dbfd9770cc3b1f232025df53b628 along with 657453d34914832f6e8012fbd69200e9680bd9ff

again, i suggest use `xfs`. for example, see https://github.com/ceph/ceph/blob/master/qa/suites/rados/objectstore/ceph_objectstore_tool.yaml#L10